modules/extreme_year.py

```diff
@@ -0,0 +1,164 @@
+"""
+This module will return the highest and lowest temperatures of a
+given year along with the most humid day.
+"""
+
+from modules.utils import read_data
+
+
+def get_highest_temperature(line):
+    """
+    Returns the highest temperature from the line read of a weather file
+    Args:
+        line(list): a list of strings containing different fields at different index
+                    please have a look at any weatherfile for more clarity
+    Returns:
+         highest_temperature(int): Value stored at index 1 of line is highest temperature
+    """
+    highest_temperature = int(line[1]) if line[1] != "" else -1000
```

Comment:
If there is by any chance a non-digit entry in the file e.g. `a` the function will crash. We should probably do this with `try: except:

```suggestion
try:
    return int(line[1])
except:
    pass
```

A hint, Try to catch the specific exception and identify if we should raise it or just pass it.

**Note:** you can do the similar changes in other utility functions as well. e.g. `get_lowest_temperature` & `get_max_humidity `

Reply:
done